docs/variants/msi_z690/montage.md

```diff
@@ -0,0 +1,123 @@
+# Dasharo compatible with MSI PRO Z690-A DDR4 - montage
+
+## Intro
+
+This document describes the montage procedure of the MSI PRO Z690-A DDR4 with
+specified components in [requirements](#requirements).
+
+## Requirements
+
+| Part         | Model Name                                                      | Image                                   |
+|:-------------|:---------------------------------------------------------------:|:---------------------------------------:|
+| Motherboard  | MSI PRO Z-690A DDR4                                             |![Motherboard](images/motherboard.jpg)   |
+| CPU          | Intel Core i5-12600K, 3.7G                                      |![CPU](images/cpu.jpg)                   |
+| Cooling      | Noctua CPU NH-U12S Redux (w/ Noctua NM-i17xx-MP78 Mounting Kit) |![Cooler](images/cpu-cooler.jpg)         |
+| RAM          | Kingston Fury Beast, DDR4, 4*8GB (32GB Total), 3600MHz, CL17    |![RAM](images/memory.jpg)                |
+| Power Supply | Seasonic Focus PX 750W 80 Plus Platinum                         |![Power Supply](images/power-supply.jpg) |
+| Storage      | SSD Intel 670p 512 GB M.2 2280 PCI-E x4 Gen3 NVMe               |![Storage](images/storage.jpg)           |
+| Enclosure    | SilentiumPC Armis AR1                                           |![Enclosure](images/enclosure.jpg)       |
+
+Additionally, you will need a regular Phillips screwdriver and thermal paste.
+Remember that make all connections in the grounding strap.
+
+## Montage
```

Comment:
Device assembly in my opinion sounds better.

Reply:
renamed